release-notes/5.0/api-diff/netcoreapp3.1/.Net/standalone/5.0-standalone_System.Json.md

```diff
@@ -0,0 +1,262 @@
+# System.Json
+
+``` diff
+ {
+ {
+-    namespace System.Json {
+ {
+-        public class JsonArray : JsonValue, ICollection<JsonValue>, IEnumerable, IEnumerable<JsonValue>, IList<JsonValue> {
```

Comment:
I don't think these types ever shipped in a stable package version...

Reply:
It seems like we shipped them in 3.1, but stopped shipping in 5.0, we should remove this.
But these apis are present in 4.7.0